src/one_dragon_qt/widgets/setting_card/app_run_card.py

```
@@ -31,14 +27,17 @@ def __init__(self, app: Application, switch_on: bool = False,
 
+        # 先初始化主基类
         MultiPushSettingCard.__init__(
             self,
-            btn_list=[self.move_up_btn, self.run_btn, self.switch_btn],
+            btn_list=[self.run_btn, self.switch_btn],
             icon=FluentIcon.GAME,
             title=self.app.op_name,
             parent=parent,
         )
 
+        # 拖拽逻辑由容器级 mixin 负责，此处不再处理
```

Ruff (0.13.3)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)